Pre-Sprint Email (#57)
thoughtbot/design-sprint-guide: resources/pre-sprint-setup-email.md

---
title: Pre-Sprint Setup Email
---

Hi _________,

We're excited to start your design sprint!

To prepare for it, it would be great to have:

> (1) a list of websites similar to parts of what we want to create, as well as other websites with aspects we may wish to emulate

Reviewer: ```suggestion
(1) a list of competitors' and peers' websites similar to parts of what we want to create, as well as other resources with aspects we may wish to emulate.
```

(2) information about each of your key customer types: who they are, their stories, and how they feel about the problem we are working to solve. If there's documentation from interviews you conducted with them, we'd love to see that too.

(3) Any existing materials you already have on hand, such as user stories, wireframes, or prototypes. No need to flesh these out more than you already have, but they'd be great background. Note that what we end up with at the end of the sprint may be different.

(4) Any other background materials you already have.

> (5) We'll start each day at 9am and end at approximately 4pm. This gives our team time to debrief and prep for the next day's session.

Reviewer: ```suggestion
(5) We'll start each day at {time} and end at approximately {time}. This gives our team time to debrief and prep for the next day's session. We will send a more detailed agenda for each session soon.
```

> (6) For {date}, we'll want to schedule time with users in the morning and early afternoon. Six users for 30 minutes each would be ideal. If you identify the people you want to bring in, we'd be happy to schedule them, or you could schedule the times directly if that's easier. We'll want to finish by 2:30pm to prepare findings for a 3:30pm discussion that day that will end the Design Sprint. We'll have each participant use the prototype we built and ask them questions to get their feedback.

Reviewer: ```suggestion
(6) For {date}, we'll want to schedule time with users in the morning and early afternoon. Six users for {minutes/hour} each would be ideal. If you identify the people you want to bring in, we'd be happy to schedule them, or you could schedule the times directly if that's easier. We'll want to finish by 2:30pm to prepare findings for a 3:30pm discussion that day that will end the Design Sprint. We'll have each participant use the prototype we built and ask them questions to get their feedback.
```

Reviewer: >  We'll want to finish by 2:30pm to prepare findings for a 3:30pm discussion that day that will end the Design Sprint. We'll have each participant use the prototype we built and ask them questions to get their feedback.

Do we still do this? I often save that for a different day.

Author: Good point, I've revised to remove the mention of an early wrap up time.
